modules/paraflare/preInitClient.sqf

```diff
@@ -1,9 +1,7 @@
-#include "script_component.hpp"
-
-private _version = 0.1;
-
-["Paraflares", "Paraflares, old style.", "TinfoilHate &amp; StatusRed", _version] call EFUNC(FW,RegisterModule);
-
-GVAR(flareAngle) = 90;
-
-["paraflare_evt_flareLight", FUNC(flareLight)] call CBA_fnc_addEventHandler;
+#include "script_component.hpp"
+
+private _version = 0.1;
+
+["Paraflares", "Paraflares, old style.", "TinfoilHate &amp; StatusRed", _version] call EFUNC(FW,RegisterModule);

```

Comment:
All your name here pal.

Reply:
If I click through to your branch I see the update, but still don't see the update in this PR? :S